README.md

```diff
@@ -20,6 +20,16 @@ The AWS Serverless Application Model (SAM) CLI is an open-source CLI tool that h
 - **Tail CloudWatch logs and X-Ray traces** with `sam log` and `sam trace`
 
 
+## Recent blogposts and workshops
+
+* **Develop Node projects with SAM CLI using esbuild (Beta)** - and use SAM Accelerate on Typescript projects. [Read blogpost here](https://s12d.com/5Aa6u0o7).
+
+* **Speed up development with SAM Accelerate (Beta)** - quickly test your changes in the cloud. [Read blogpost here](https://s12d.com/WeMib4nf).
```

Comment:
Will pause the merge until a revised version of this article is published

Reply:
I suppose we can remove "(Beta)" since Accelerate it's now GA?

```suggestion
* **Speed up development with SAM Accelerate** - quickly test your changes in the cloud. [Read blogpost here](https://s12d.com/WeMib4nf).
```

Reply:
fixed!